Linker error while compiling cryptography-rust

I don't know why this issue plagued me and what is the culprit.
I am trying Windows build for latest pyca cryptography 36 dev1 and got linker error while compiling cryptography-rust:

"C:\Python37\cryptography-main\src\rust\target\i686-pc-windows-gnu\release\deps\cryptography_rust.cryptography_rust.8yeboqs1-cgu.12.rcgu.o:cryptography_rust.: undefined reference to _imp___ZN3pem11ASCII_ARMOR17hda6e74d90990760eE
collect2.exe: error: ld returned 1 exit status"

Missing function should obviously be named _ZN3pem11ASCII_ARMOR17hda6e74d90990760eE and linked from pem library.
But why it got erroneous _imp__ prefix? No idea.

I'm using 32-bit environment with Python 3.7 / Rust 1.48 (i686-pc-windows-gnu) / PEM 1.0.0

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,3 +1,7 @@
+# 1.0.1
+
+ - hide the ASCII_ARMOR symbol to work around a linking issue with 32-bit windows builds
+
 # 1.0
 
  - `pem::parse_many` now returns a `Result<Vec<Pem>>` instead of a `Vec<Pem>` that silently discarded invalid sections.

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -116,9 +116,14 @@ const REGEX_STR: &str =
 /// The line length for PEM encoding
 const LINE_WRAP: usize = 64;
 
-static ASCII_ARMOR: Lazy<Regex> = Lazy::new(|| {
-    Regex::new(REGEX_STR).unwrap()
-});
+
+fn ascii_armor() -> &'static Regex {
+    static ASCII_ARMOR: Lazy<Regex> = Lazy::new(|| {
+        Regex::new(REGEX_STR).unwrap()
+    });
+
+    &ASCII_ARMOR
+}
 
 /// Enum describing line endings
 #[derive(Debug, Clone, Copy)]
@@ -242,7 +247,7 @@ impl Pem {
 ///  assert_eq!(pem.tag, "RSA PRIVATE KEY");
 /// ```
 pub fn parse<B: AsRef<[u8]>>(input: B) -> Result<Pem> {
-    ASCII_ARMOR
+    ascii_armor()
         .captures(&input.as_ref())
         .ok_or_else(|| PemError::MalformedFraming)
         .and_then(Pem::new_from_captures)
@@ -319,7 +324,7 @@ pub fn parse<B: AsRef<[u8]>>(input: B) -> Result<Pem> {
 /// ```
 pub fn parse_many<B: AsRef<[u8]>>(input: B) -> Result<Vec<Pem>> {
     // Each time our regex matches a PEM section, we need to decode it.
-    ASCII_ARMOR
+    ascii_armor()
         .captures_iter(&input.as_ref())
         .map(|caps| Pem::new_from_captures(caps))
         .collect()

